tool_collections/samtools/samtools_stats/samtools_stats.xml

```diff
@@ -1,8 +1,12 @@
-<tool id="samtools_stats" name="Generate statistics" version="1.0.1">
-    <description>for a BAM or SAM file</description>
+<tool id="samtools_stats" name="Stats" version="1.1">
+    <description>generate statistics for BAM dataset</description>
     <macros>
     <import>macros.xml</import>
   </macros>
+    
+    <!-- the code file below should be enabled with samtools -I bug is fixed -->
+    <!--<code file="samtools_stats_options.py"/>-->
+
```

Comment:
@nekrut can you provide some more infos here? If we depend on a strict samtools version, is this needed or not?